migration/v0.8.x-v0.9.x/upgrade-apps.md

```diff
@@ -3,30 +3,42 @@
 
 You will need to follow these steps in order to upgrade each Compliant Kubernetes environment from v0.8.x to v0.9.0. The instructions assume that you work from the root of the `compliantkubernetes-apps` repository, has the latest changes already pulled and configured `CK8S_CONFIG_PATH` correctly.
 
-1. This is an optional step. The default resource requests for Harbor pods have been updated. To get these new values you must remove the current values in your config (`resources:` yaml blocks under `harbor:`) from `sc-config.yaml` before running `./bin/ck8s init` in the next step.
+1. Checkout the new release: `git checkout v0.9.0`.
 
-2. Run init to get new defaults: `./bin/ck8s init`
+2. Update helm to v3.5.0.
+
+3. Optional
+ The default resource requests for Harbor pods have been updated.
+ To get these new values you must remove the current values in your config (`resources:` yaml blocks under `harbor:`) from `sc-config.yaml` before running `./bin/ck8s init` in the next step.
+
+4. Replace the config option `global.environmentName` with `global.clusterName` run `./migration/v0.8.x-v0.9.x/migrate-config.sh`.
```

Comment:
It seems like the script is an empty file, so I don't think we need to run it.

Reply:
I have no idea why github says it's empty, it's not :D

Reply:
Hmm, maybe it's because the diff is only the mode and not a change in the file